src/flb_io_tls.c

```diff
@@ -71,6 +72,51 @@ static inline int io_tls_event_switch(struct flb_upstream_conn *u_conn,
     return 0;
 }
 
+#ifdef _MSC_VER
+static int flb_tls_load_system_cert(struct flb_tls_context *ctx)
+{
+    int ret;
+    HANDLE h;
+    PCCERT_CONTEXT cert = NULL;
+
+    h = CertOpenSystemStoreA(NULL, "Root");
+    if (h == NULL) {
+        flb_error("[TLS] Cannot open cert store: %i", GetLastError());
```

Comment:
Can we obtain and print detail error message with [`FormatMessageA`](https://docs.microsoft.com/en-us/windows/desktop/api/winbase/nf-winbase-formatmessagea) here?
(It is not urgent task, just a recommendation for future improving.)

Reply:
Thanks. I'll try to include the logging improvement in later patches.